content/chainguard/chainguard-images/authenticating.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Authenticating to Chainguard Registry"
```

Comment:
If we are planning on having more than one doc related to Chainguard Registry, I would recommend creating a `/registry` directory and have `authenticating.md` live within that